circuits/cpp/src/aztec3/circuits/abis/c_bind.cpp

```diff
@@ -320,6 +320,33 @@ WASM_EXPORT void abis__compute_contract_address(uint8_t const* point_data_buf,
     NT::fr::serialize_to_buffer(contract_address, output);
 }
 
+/**
+ * @brief Compute a contract address from its partial contract address
+ * This is a WASM-export that can be called from Typescript.
+ *
+ * @details Computes a contract address by hashing the deployers public key along with the previously computed partial
+ * address Return the serialized results in the `output` buffer.
+ *
+ * @param point_data_buf point data struct as a buffer of bytes
+ * @param partial_address_data_buf partial contract address
+ * @param output buffer that will contain the output. The serialized contract address.
+ */
+WASM_EXPORT void abis__compute_contract_address_from_partial(uint8_t const* point_data_buf,
```

Comment:
Im pretty sure this could be replaced with the `MSGPACK` variant as just 
```cpp
CBIND(abis__compute_contract_address_from_partial, aztec3::circuits::compute_contract_address_from_partial);
```